dataeng/jobs/createJobs.groovy

```diff
@@ -67,54 +68,7 @@ try {
 }
 
 def taskMap = [
-    ANALYTICS_EMAIL_OPTIN_JOB: AnalyticsEmailOptinJob,
-    ANALYTICS_EXPORTER_JOB: AnalyticsExporterJob,
-    USER_ACTIVITY_JOB: UserActivityJob,
-    VIDEO_TIMELINE_JOB: VideoTimelineJob,
-    USER_LOCATION_BY_COURSE_JOB: UserLocationByCourseJob,
-    ENROLLMENT_JOB: EnrollmentJob,
-    MODULE_ENGAGEMENT_JOB: ModuleEngagementJob,
-    DATABASE_EXPORT_COURSEWARE_STUDENTMODULE_JOB: DatabaseExportCoursewareStudentmoduleJob,
-    ANSWER_DISTRIBUTION_JOB: AnswerDistributionJob,
-    CYBERSOURCE_PULL_JOB: CybersourcePullJob,
-    PAYMENTS_VALIDATION_JOB: PaymentsValidationJob,
-    FINANCE_REPORT_JOB: FinanceReportJob,
-    BIGQUERY_REPLICA_IMPORT_JOB: BigqueryReplicaImportJob,
-    EVENT_EXPORT_INCREMENTAL_JOB: EventExportIncrementalJob,
-    COURSEWARE_LINKS_CLICKED_JOB: CoursewareLinksClickedJob,
-    EVENT_TYPE_DISTRIBUTION_JOB: EventTypeDistributionJob,
-    GENERATE_WAREHOUSE_DOCS_JOB: GenerateWarehouseDocsJob,
-    SQL_SCRIPTS_JOB: SqlScriptsJob,
-    SINGLE_SQL_SCRIPT_JOB: SingleSqlScriptJob,
-    LOAD_AFFILIATE_WINDOW_JOB: LoadAffiliateWindowWarehouseJob,
-    LOAD_WAREHOUSE_VERTICA_JOB: LoadWarehouseVerticaJob,
-    LOAD_WAREHOUSE_SNOWFLAKE_JOB: LoadWarehouseSnowflakeJob,
-    LOAD_WAREHOUSE_BIGQUERY_JOB: LoadWarehouseBigQueryJob,
-    LOAD_EVENTS_TO_S3_JOB: LoadEventsToS3Job,
-    LOAD_EVENTS_TO_VERTICA_JOB: LoadEventsToVerticaJob,
-    LOAD_JSON_EVENTS_TO_S3_JOB: LoadJsonEventsToS3Job,
-    LOAD_JSON_EVENTS_TO_BIGQUERY_JOB: LoadJsonEventsToBigqueryJob,
-    VERTICA_REPLICA_IMPORT_JOB: VerticaReplicaImportJob,
-    VERTICA_TO_BIGQUERY_SCHEMA_COPY_JOB: VerticaToBigquerySchemaCopyJob,
-    BACKUP_VERTICA_JOB: BackupVerticaJob,
-    TOTAL_EVENTS_DAILY_REPORT_JOB: TotalEventsDailyReportJob,
-    JENKINS_BACKUP_JOB: JenkinsBackupJob,
-    EVENT_EXPORT_INCREMENTAL_LARGE_JOB: EventExportIncrementalLargeJob,
-    LOAD_COURSE_STRUCTURE_JOB: LoadCourseStructureJob,
-    ENTERPRISE_JOB: EnterpriseJob,
-    TERMINATE_CLUSTER_JOB: TerminateClusterJob,
-    EMR_COST_REPORTER_JOB: EmrCostReporterJob,
-    VERTICA_DISK_USAGE_MONITOR_JOB: VerticaDiskUsageMonitorJob,
-    UPDATE_USERS_JOB: UpdateUsersJob,
-    ENROLLMENT_VALIDATION_EVENTS_JOB: EnrollmentValidationEventsJob,
-    DEPLOY_CLUSTER_JOB: DeployClusterJob,
-    LOAD_INSIGHTS_TO_VERTICA_JOB: LoadInsightsToVerticaJob,
-    LOAD_GOOGLE_ANALYTICS_PERMISSIONS_JOB: LoadGoogleAnalyticsPermissionsJob,
-    AGGREGATE_DAILY_TRACKING_LOGS_JOB: AggregateDailyTrackingLogsJob,
-    MONITOR_BIGQUERY_EVENT_LOADING_JOB: MonitorBigqueryEventLoadingJob,
-    LOAD_VERTICA_SCHEMA_TO_SNOWFLAKE_JOB: LoadVerticaSchemaToSnowflakeJob,
-    LOAD_GOOGLE_SPREADSHEET_TO_WAREHOUSE_JOB: LoadGoogleSpreadsheetToWarehouseJob,
-    SNOWFLAKE_REPLICA_IMPORT_JOB: SnowflakeReplicaImportJob,
+    LOAD_PAYPAL_CASEREPORT_TO_VERTICA_JOB: PayPalCaseReportLoadJob,
```

Comment:
I think you did not mean to leave these deleted?

Reply:
Yes i know, Just wanted to seed the job first. 
I'll add these files again before merging it.